v63004/inref/gtm5730.m

```diff
@@ -0,0 +1,21 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+gtm5730()
+	 TSTART
+	 FOR i=1:1:2000 set ^x(i)=i ;
```

Comment:
Since we do the 2000 SETs inside a TSTART/TCOMMIT, we are doing just ONE transaction/sequence-number. It is better to move the FOR loop outside the TSTART/TCOMMIT logic. And do 1000 of TSTART/TCOMMIT and 1000 without TSTART/TCOMMIT. That way we can expect to see TCOM as well as SET records in the reference file. In addition, it would be better to see KILL record too in the reference file.

Reply:
I've split the .m file into 2 forloops for 1000 individual TSTART/TCOMMIT transactions and 1000 regular transactions respectively. ^x is killed at the end of the m program to generate the kill type record.